docs/comparisons.md

```diff
@@ -85,3 +85,15 @@ pip-run poetry -- -m poetry --help
 
 Example interaction:
 None.
+
+## pipx vs fades
+
+[fades](https://github.com/PyAr/fades) is a tool to run **individual** Python scripts inside automatically provisioned virtualenvs with their dependencies installed.
+
+* Both [fades](https://github.com/PyAr/fades#how-to-mark-the-dependencies-to-be-installed) and [pipx](https://github.com/pypa/pipx/pull/916) allow specifying a script's dependencies in specially formatted comments, but the exact syntax differs (see also [PEP 722](https://peps.python.org/pep-0722/) which seeks to standardize such syntax).
```

Comment:
I linked the PR here instead of the docs because I didn't notice that pipx already has documentation on this which just isn't published yet because there hasn't been a release since. Will update.

Reply:
Updated